src/services/usersServices.js

```diff
@@ -0,0 +1,45 @@
+'use strict';
+
+const { unicId } = require('../helpers/helpers');
+
+let users = [];
+
+function getAll() {
+  return users;
+};
+
+function getById(id) {
+  const userFound = users.find(user => user.id === id);

```

Comment:
check your type because now i think user.id its number but id its string 
and check it out in all other func where you use id

Reply:
not fixed, try to figure out with type of your id, and check it in all places, i think its help you to pass tests

Reply:
not fixed